nodebuilder/node_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	collectormetricpb "go.opentelemetry.io/proto/otlp/collector/metrics/v1"
 	"google.golang.org/protobuf/proto"
 
+	"github.com/celestiaorg/celestia-node/cmd/version"
```

Comment:
Nodebuilder should not depend on cmd. I don't know where to put BuildInfo yet to fix this and open to suggestions

Reply:
🤔 . I think then that the best way is to revert changes related to `nodebuilder`: to have own buildInfo in `nodebuilder` package + cast from the `version.BuildInfo` to the `nodebuilder.buildInfo`

Reply:
Sounds good to me, or even avoid the BuildInfo inside of `version`

Reply:
I thought about it, but imo it looks a bit nicer when we return struct